code/modules/clothing/masks/gasmask.dm

```diff
@@ -15,6 +15,17 @@
 	siemens_coefficient = 0.9
 	var/gas_filter_strength = 1			//For gas mask filters
 	var/list/filtered_gases = list(/datum/reagent/toxin/phoron, "sleeping_agent", "carbon_dioxide")
+	var/breathy = 1		///Whether or not this plays a sound on equip
```

Comment:
TRUE / FALSE
also, you know that this is not how we autodoc

Reply:
Done.